RibonDAO/interface
# Tech 299: Events refactor

# Description

- created a lib for log the events, this lib would call the firebase and later mixpanel and amplitude.
- modal refactor: changed the buttons attributes that were separated for only one attribute that is an object with all button props
- added new events for the donation flow(check sheet on jira task, only the "DoaçãoTicket_atualizado" tab was updated)
- removed old events for the donation flow
- added an automatic way on routes to log every page view

## Type of change

Please delete options that are not relevant.


## How to test

Put here all the things need to test this PR and verify your changes, also list any relevant details for tests (eg: have a wallet)

- Check if the events from the sheet tab(DoaçãoTicket_atualizado) are being logged through console on browser
- Check if any modal was changed(by style or functionality)
